src/extractcode/extract_directory.py

```diff
@@ -0,0 +1,131 @@
+#
+# Copyright (c) 2015 nexB Inc. and others. All rights reserved.
```

Comment:
I guess we are no longer on 2015, are we?
Be careful to not copy extra things around without checking first.